build.gradle

```diff
@@ -71,6 +72,11 @@ static def jniLibOsClassifier() {
     throw new RuntimeException("platform not supported: " + System.getProperty("os.name"))
 }
 
+static def jniLibArchClassifier() {
+    def target = rustTargetTriple() ?: ""
+    return target ? "_" + target.split("-")[0] : ""
```

Comment:
This project will support multiple target architectures for one operating system after this PR. I think for consistency, we should change all shared objects to have consistent naming convention that includes architecture as well, not just for aarch64 + macos.

Reply:
Makes sense - I was going for minimal changes, but it's probably best to be consistent.